R/unstash.R

```diff
@@ -19,13 +20,13 @@
 #' }
 #'
 #' @export unstash
-unstash <- function(var) {
+unstash <- function(var, verbose=TRUE) {
```

Comment:
Can you put a space between the variables, values and equal sign? `verbose = TRUE`. Thanks!